client/components/Footer.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+
+/**
+ *
+ *
+ * @class Footer
+ * @extends {React.Component}
+ */
+class Footer extends React.Component {
+
+  /**
+   *
+   *
+   * @returns
+   *
+   * @memberOf Footer
+   */
+  render() {
+    return (
+      <footer className="page-footer blue-grey darken-4 footer">
+        <div className="footer-copyright">
+          <div className="container">
+            © 2017 DocStar | All Rights Reserved | Document Management System
+            <a className="grey-text text-lighten-4 right" href="#!">H
+              elp Line (+234-672391083, +299-031452)</a>
+          </div>
+        </div>
+      </footer>
+
+    );
+  }
+}
+
+
+export default connect(null)(Footer);
```

Comment:
I don't think there is any need to use `connect()` here since you are not using it